tutorials/01-conda_astropy_install.rst

```diff
@@ -0,0 +1,139 @@
+
+Pre-requisites
+--------------
+
+**Mac** Have the latest version of `Xcode
+developer <https://developer.apple.com/xcode/>`__ tools installed
+
+**Windows** Be able to access a terminal, either through a `Linux
+subsystem <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__
+or by installing the `Linux bash
+shell <https://www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/>`__
+
+Step 1: Download Anaconda
+-------------------------
+
+The Anaconda Python distribution can be downloaded from
+https://www.anaconda.com/download
+
+-  The scientific computing community is now using Python 3 as a
+   default.
+
+**But my code only runs in Python 2** Please see the next tutorial: *An
+Astropy User's Guide to Managing Conda Environments*
+
+-  When the download is finished, click on the package and follow the
+   installation instructions.
+
+-  Open a terminal window to check that the Anaconda installation will
+   work for you:
+
+   ::
+
+       which conda
+
+   should return something like ``/anaconda3/bin/conda`` and
+
+   ::
+
+       which python
+
+   should return a Python path that is in the same directory as
+   Anaconda: ``/anaconda3/bin/python``
+
+Step 2: Install core packages
+-----------------------------
+
+The default Anaconda installation comes with many packages that
+astronomers use frequently: *numpy*, *scipy*, and *matplotlib*
+
+We can use the ``conda install`` command to install everything else we
+need. Anaconda will automatically check, update, and install any python
+packages that your desired package depends on.
+
+-  Below, we give an example of installing astropy along with some common 
+   scientific, statistical, and visualization packages. You can install them all
+   individually or in one line:
+
+::
+
+    conda install astropy matplotlib scikit-learn pandas
+
+Step 3: Install affiliated packages
+-----------------------------------
+
+Many `Astropy affiliated
+packages <https://www.astropy.org/affiliated/>`__ can be found on
+*astroconda* channel, maintained by AURA and STScI. To add this channel
+to Anaconda's package search list, run the following command:
+
+::
+
+    conda config --add channels http://ssb.stsci.edu/astroconda
+
+Some astronomical packages are also available in the *conda-forge*
+channel. There is no wrong choice between installing a package from
+*astroconda* versus *conda-forge*. However, a package that is available
+in the *astroconda* channel may not be available in *conda-forge*.
+
+To see what channels you have available:
+
+::
+
+    conda config --show channels
+
+`More information on managing channels in
+Anaconda <https://conda.io/docs/user-guide/tasks/manage-channels.html>`__
+is available on the main documentation pages.
+
+-  Here's an example for downloading a few commonly used Astropy
+   affiliated packages, directly from the *astroconda* channel:
+
+   ::
+
+       conda install -c astroconda astroquery photutils specutils
```

Comment:
for astroquery we've switched to continuous delivery, that currently means that the preferred way of install is 
`pip install --pre astroquery`. The conda version by definition won't be the latest release (we may change to that, but the machinery is not yet in motion for it)

Reply:
Mixing `conda install` with `pip` will confuse many new users. As long as the conda releases are stable and ongoing, let's stick to that.

People who need the latest versions on a regular basis are probably developing. How about adding the pip install line to the "Setting up a developer version of Astropy", instead?

Reply:
No, the idea here is that these are stable releases. For astroquery it isn't true that the latest "stable" is more reliable than these auto deployed releases. 
We'll eventually fix the version numbering, so normal pip install (without ``--pre``) will be able to pick up the releases, but I'm not sure we can do a continuous supply of conda packaging, too.

The developer version is a completely different question, one has that for having access to the git tree, local docs, etc.

Reply:
also, users will eventually need pip even when using conda as not all packages are available on conda

Reply:
My solution here is to remove astroquery from this example. I've added notes at the end of the document telling users to use pip for managing astroquery.